manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -89,14 +89,22 @@ Then the source of the add-on package will be checked out into the ``src`` folde
 
 .. note::
 
-    You can also add Products.PDBDebugMode and plone.reload to your development-tools:
+    You can also add ``Products.PDBDebugMode`` and ``plone.reload`` to your development-tools. For ``Products.PDBDebugMode`` you currently need a source-checkout though.
 
```

Comment:
Please follow our style guide.
One sentence per line.
https://docs.plone.org/about/contributing/documentation_styleguide.html

Thanks!

Reply:
Fixed in 9f40573.